cmd/client/BUILD

```diff
@@ -11,7 +11,6 @@ go_library(
         "report.go",
         "util.go",
     ],
-    visibility = ["//visibility:private"],
```

Comment:
can we limit the visibility to something like `//cmd:__subpackages__`or similar?